arcavios/scooze
# add support for docker-based mongo

To test: 

- pip install scooze
- install docker desktop and make sure it's running
- run `scooze setup docker` to pull and initailize a docker-based mongo instance
- populate that DB as normal with `scooze load-cards oracle`

Limitations: 
- Needs docker installed and running
- Needs port 27017 to be free on the host machine
